app/assets/javascripts/table-sort.es6

```diff
@@ -2,6 +2,9 @@ $(document).ready(function() {
   if ($('table.sortable')) {
     let table;
     $('th.sort')
+      .addClass("js-sortable")
+        //this class used to style as links, if JS not enabled, titles won't look clickable.
```

Comment:
Very minor suggestion - but usually I'd expect a comment to be above the item it's concerning. So:
```javascript
//this class used to style as links, if JS not enabled, titles won't look clickable.
.addClass("js-sortable")
```
Or to make it really clear put it on the same line:
```javascript
.addClass("js-sortable") //this class used to style as links, if JS not enabled, titles won't look clickable.
```